### DISCUSSION TITLE
We have discussions tab now!

### DISCUSSION THREAD

**Discussion by LuanderFarias** [betterbrowser/arcfox#101] [Announcements]
## 👋 Discussions!
  We’re using Discussions now because i guess it makes sense :D
  * Ask questions you’re wondering about.
  * Engage with other community members.
  * Share ideas.

**Comment by whitererr** [comment]
Will you ever shut down/stop working on this project? I'm debating whether to use this or arcWTF.

**Reply by LuanderFarias** [reply]
no. the project might be dead from time to time but i never plan on stopping or shutting it down. unfortunately due to my work schedule here on brazil i dont have much time to work on the project, but ill never stop working on it.
